apf-ingester/apf_Ingester.py

```diff
@@ -0,0 +1,264 @@
+import re
+import os
+import json
+import shutil
+import tempfile
+import pathlib
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+from typing import Dict, List, Optional
+
+
+def cleanup_source_document(sgm_path: str) -> None:
+    """Removes CRLF EOLs and in-text markup  in the file to preserve string offsets used for
+       annotation (as the ACE annotation offsets use '\n' EOLs and ignore markup)
+
+    Args:
+        sgm_path (str): String path of the .sgm file
+
+    One of sgm_string or sgm_path must be specified
+
+    Returns:
+        None
+
+    """
+    with open(sgm_path, 'r') as sgm_file:

```

Comment:
If you make `sgm_path` a `Path`, you can just do `sgm_string = sgm_path.read_text(encoding="utf-8")`